### PR TITLE
Add configuration for volcano, a batch system on k8s. [BA-6011]

### PR DESCRIPTION
**Pull request by jiangkaihua**
Volcano is a batch system running high performance workloads on Kubernetes, which provides scheduling, job management, and other mechanisms k8s missed. This is an example configure for cromwell to run on volcano clusters.

### CODE REVIEW

File: docs/backends/volcano.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+The following configuration can be used as a base to allow Cromwell to interact with a [volcano](https://volcano.sh/) cluster and dispatch jobs to it:
+
+```hocon
+VK {
```
Comment (k82cn) [inline docs/backends/volcano.md]: s/VK/Volcano

File: docs/backends/volcano.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+The following configuration can be used as a base to allow Cromwell to interact with a [volcano](https://volcano.sh/) cluster and dispatch jobs to it:
```
Comment (k82cn) [inline docs/backends/volcano.md]: let's use github address of Volcano.

**Comment by jiangkaihua** [comment]
> Looks good to me, once @cjllanwarne's comments are addressed. Thank you!
> Out of curiosity, what kind of cluster/tooling/data processing work are you using at your site?

Actually, we're trying to run gene sequencing tasks using cromwell, meanwhile adopting Volcano as  scheduler for our kubernetes clusters.

**Comment by aednichols** [comment]
@jiangkaihua your contribution has merged, thank you for stopping by our repo!

**Comment by luyang93** [comment]
@jiangkaihua I want to knwo how does cromwell generate a yaml file for volcano?
Cromwell now genrates a bash script file when `actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"`, while vcctl job run just accept a yaml file.
Do I need to implement a volcano backend?

**Comment by fferrann** [comment]
> @jiangkaihua I want to knwo how does cromwell generate a yaml file for volcano?
> Cromwell now genrates a bash script file when `actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"`, while vcctl job run just accept a yaml file.
> Do I need to implement a volcano backend?

Did you end up figuring this out? Has somebody implemented a Volcano backend?

**Pull request merged**